packages/scandipwa/src/component/ProductAttributeValue/ProductAttributeValue.component.js

```diff
@@ -57,12 +59,25 @@ export class ProductAttributeValue extends PureComponent {
     }
 
     getOptionLabel(value) {
-        const { attribute: { attribute_options } } = this.props;
+        const {
+            attribute: {
+                attribute_options
+            },
+            showProductCount
+        } = this.props;
 
         if (attribute_options) {
             const optionValues = attribute_options[value];
             if (optionValues) {
-                return optionValues;
+                if (!showProductCount) {
+                    return optionValues;
+                }
+
+                const { label, count = 0 } = optionValues;
+                return {
+                    ...optionValues,
+                    label: `${label} (${count})`
```

Comment:
How this will work with swatches? Won't show them at all?

Reply:
@lianastaskevica the swatch item count won't be shown, is this expected?

Reply:
On swatches label is visible only on hover.